Makefile

```diff
@@ -38,10 +41,16 @@ endif
 
 test:
 	# testing..
-	CGO_ENABLED=1 go test -race -cover $(TEST_PKGS)
+	go get github.com/etcd-io/gofail
```

Comment:
use another place to install gofail

Reply:
use another place to run gofail test, not with the common tests.

Reply:
Actually, I use TiDB's makefile as a reference here.

Reply:
I think this should be installed (in both projects) with retool. Add it to `./hack/retool-install.sh` and then commit tools.json. And then it can be removed from Gopkg.toml since it isn't actually a build dependency.

Reply:
@gregwebs Since we need import gofail when running the time fallback test, I don't think it can be removed from `Gopkg.toml`. The reason is `dep` will add the dependency according to `import`. Am i right?

Reply:
Oh, right, I didn't realize it was being imported in tests also.